### DISCUSSION TITLE
[测试]尝试上传一个爱莉的视屏

### DISCUSSION THREAD

**Discussion by Elysia-Ego-093** [share121/inter-knot#1030] [其他]
![Screenshot_2024-08-10-21-24-34-030_com miui mediaviewer](https://github.com/user-attachments/assets/6794c0a9-e805-40cc-9f84-884f5e4d3e02)

https://github.com/user-attachments/assets/5eb4c2e2-8369-46fa-a994-1847f0464b6f

有点糊，github限制只能10MB以内，只能压缩画质了

**Comment by capacitor1** [comment]
摘下眼睛，提升画质doge

**Comment by themagicmoon** [comment]
能看就行🙃

**Comment by Qiling-XwX** [comment]
![OIP-C](https://github.com/user-attachments/assets/3803d995-4e69-4372-8f7c-5ec91652de6d)

**Comment by xibozhi** [comment]
纯萌新，以后绳网的上传限制会提升吗

**Reply by jenemi** [reply]
可以，只要脱离github就行

**Reply by mhpmc** [reply]
不脱离github的话10mb是被锁死的
